### PR TITLE
Various UI improvements

### PR DESCRIPTION
**Pull request by kathaypacific**
- improve spaces/border radius consistency
- make the bottom states more intuitive (i think having zap in or zap out disabled in the expanded manage view was not intuitive, as you couldn't switch back and forth between the views without collapsing the whole thing first)
- improve the dropdown animation smoothness (still not perfect but good enough)
- include loading state for TVL and staked value so that the UI does not jump around

Screenshot 2022-03-12 at 08 26 24

Screenshot 2022-03-12 at 08 29 09

### CODE REVIEW

File: src/components/earn/PoolCard.tsx [hunk 1]
```diff
@@ -151,17 +152,25 @@ export const PoolCard: React.FC<Props> = ({ compoundBotSummary }: Props) => {
   // })
 
   const [zapType, setZapType] = useState<ZapType | null>(null)
+  // TODO: show zap completed state rather than just collapsing the whole box
```
Comment (cajubelt) [inline src/components/earn/PoolCard.tsx:155]: should this be an issue?

File: src/components/earn/PoolCard.tsx [hunk 2]
```diff
@@ -322,49 +345,46 @@ export const PoolCard: React.FC<Props> = ({ compoundBotSummary }: Props) => {
...
-                      </MaxButton>
+                      {zapOutPercentages.map((zapOutPercentage) => (
+                        <MaxButton
+                          key={zapOutPercentage}
+                          onClick={() => setZapOutPercentage(zapOutPercentage)}
+                          width="25%"
+                        >
+                          {`${zapOutPercentage}%`}
+                        </MaxButton>
+                      ))}
```
Comment (cajubelt) [inline src/components/earn/PoolCard.tsx:387]: ❤️

**Review by cajubelt — APPROVED** [review]
🥳

File: src/components/earn/PoolCard.tsx (hunk 1 above)
Comment (kathaypacific) [reply src/components/earn/PoolCard.tsx:155]: i thought it was just my OCD, it doesn't seem like a major issue (doesn't block functionality) but seeing the whole box collapse did take me by surprise. i'll create a non urgent issue for it :)

**Pull request merged**